src/dmd/dsymbolsem.d

```diff
@@ -1684,6 +1684,28 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             }
             goto Lnodecl;
         }
+        else if (pd.ident == Id.linkerdirective)
+        {
+            if (!pd.args || pd.args.dim != 1)
+                pd.error("string expected for linker directive");
+            else
+            {
+                auto se = semanticString(sc, (*pd.args)[0], "linker directive");
+                if (!se)
+                    goto Lnodecl;
+                (*pd.args)[0] = se;
+
+                if (global.params.verbose)
+                {
+                    auto directive = cast(char*)mem.xmalloc(se.len + 1);
+                    memcpy(directive, se.string, se.len);
+                    directive[se.len] = 0;
+                    message("linker directive   %s", directive);
```

Comment:
I guess this doesn't align with other verbose messages.

Reply:
I'm not sure what you mean? It's a carbon copy of the lined from pragma(lib) just above...?

Reply:
Try compiling with `-v` to see what I mean. 😉

I don't see it as a problem, the whitespace padding seems quite arbitrary anyway.

Reply:
I just made it the same as the code nearby...